main.go

```diff
@@ -34,7 +38,7 @@ func main() {
 		downstream string
 	)
 
-	flag.StringVar(&upstream, "upstream", "https://github.com/okkur/gitor.git", "specifies upstream")
+	flag.StringVar(&upstream, "upstream", upstream, "specifies upstream")
 	flag.StringVar(&branch, "branch", "master", "specifies branch")
 	flag.StringVar(&username, "username", username, "specifies username")
```

Comment:
Instead of having the variable username as default, maybe we make it clear that it's an empty string "".